views/auth/url/login.html

```diff
@@ -38,6 +38,15 @@ <h1 class="">
 							required
 						/>
 					</div>
+					{% if showUserOptIn == 'true' %}
+					<div class="form-group optin">
+						<label>
+							<input type="checkbox" id="optin-email" name="optin_email" value="email" />
+							Ik wil updates ontvangen over {{ client.name }}
```

Comment:
Het liefst heb ik deze tekst aanpasbaar via de database, waar je wel deze tekst kan gebruiken als er niks is ingesteld!